docs/cmd/kumactl/kumactl_inspect_zoneegresses.md

```diff
@@ -0,0 +1,33 @@
+## kumactl inspect zoneegresses
+
+Inspect Zone Egresses
+
+### Synopsis
+
+Inspect Zone Egresses.
+
+```
+kumactl inspect zoneegresses [flags]
+```
+
+### Options
+
+```
+  -h, --help   help for zoneegresses
+```
+
+### Options inherited from parent commands
+
+```
+      --api-timeout duration   the timeout for api calls. It includes connection time, any redirects, and reading the response body. A timeout of zero means no timeout (default 1m0s)
+      --config-file string     path to the configuration file to use
+      --log-level string       log level: one of off|info|debug (default "off")
+  -m, --mesh string            mesh to use (default "default")
```

Comment:
That's actually incorrect, we have `--mesh` both for ZoneIngress and ZoneEgress because of the flag in [root.go](https://github.com/kumahq/kuma/blob/master/app/kumactl/cmd/root.go#L78). I don't think it should be fixed here, I'll create an issue

Reply:
xref: https://github.com/kumahq/kuma/issues/3781

Reply:
I agree, let's fix it it followup PR